README.md

```diff
@@ -123,49 +148,29 @@ using System.Collections.Generic;
 
 public class MyCoolBehaviorScript : MonoBehaviour {
     void Start () {
-        Branch.initSession(CallbackWithParams);
```

Comment:
@antonargunov This is going to be a breaking build for most people since we've never recommended the callback with BUO. Can you please add this back in?